sdk/ai/Azure.AI.Inference/src/Customized/AzureAIInferenceEmbeddingGenerator.cs

```diff
@@ -0,0 +1,187 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable enable
```

Comment:
As a general pattern, we intentionally don't have nullability enabled in the Azure SDK libraries.   The rest of Inference does not enable it, so unless there's a special case that I'm overlooking, we'll want to suppress it here as well.